src/general/OptionsMenu.cs

```diff
@@ -172,6 +175,10 @@ public class OptionsMenu : ControlWithInput
         .OrderBy(i => i, StringComparer.InvariantCulture)
         .ToList();
 
+    private static readonly List<string> AudioOutputDevicesCache = AudioServer
+        .GetDeviceList().OfType<string>().Where(p => p != Constants.DEFAULT_AUDIO_OUTPUT_DEVICE_NAME)
```

Comment:
What does `p` stand for here? I would have personally selected `d` for device in this lambda.